src/main/java/Duke.java

```diff
@@ -1,10 +1,102 @@
+import duke.command.TaskList;
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
+/**
+ * The type Duke.
+ */
 public class Duke {
+
+    /**
+     * The Scanner Class.
+     */
+    static Scanner sc = new Scanner(System.in);
+
+    /**
+     * The entry point of application.
+     *
+     * @param args the input arguments
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        String greeting = "Hello! I'm Alpha Beast What can I do for you?";
+        greeting(greeting);
+        TaskList manager = new TaskList("/Users/s.f/ip/src/Data/duke.txt");
+        loop:
+        while (true) {
+            String input = sc.nextLine();
+            String[] tokens = input.split(" ");
+            String without_key = input.replace(tokens[0], "");
+
+            switch (tokens[0]) {
```

Comment:
neat and easy to understand indentations

Reply:
Thanks for the feedback tricixg.